packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-color/gux-form-field-color.scss

```diff
@@ -60,14 +60,20 @@
       align-self: auto;
       justify-content: center;
       order: 0;
-      height: var(--gse-ui-formControl-input-textfield-height);
+      width: var(--gse-ui-formControl-input-colorPicker-size);
+      height: calc(
+        var(--gse-ui-formControl-input-colorPicker-size) + 2px
+      ); // Adding 2px here so that the width and height are both 34px to make room for the border to render, otherwise it's cut off
```

Comment:
This isn't a suitable fix, because it breaks the aspect ratio of the component - it's no longer a square. You can see this in the layout tab of the CSS inspect, which is always a good place to look when you run into this kind of issue:

<img width="1367" alt="Screen Shot 2023-11-03 at 9 04 28 AM" src="https://github.com/MyPureCloud/genesys-spark/assets/604797/3890dabc-36e9-4d8c-b817-9571d226261d">

The design also calls for a 32x32 square, rather than 34x34. When you're having trouble with an element's size, it's often a good idea to check it's children. In this case, the slotted input is also out of square, which is likely part of the problem:

<img width="1367" alt="Screen Shot 2023-11-03 at 9 08 48 AM" src="https://github.com/MyPureCloud/genesys-spark/assets/604797/0eef94f2-ca2f-4697-acb4-0101c7816336">

The input is bigger than I expected as well at 32x32px instead of the 24x24 swatch from the designs. After removing the extra 2px on this line, I was able to get the correct sizing with these styles on the input:

```css
::slotted(input) {
  ...
  flex: 0 1 auto; //don't grow to the container, we want a fixed width
  width: 24px; // swatch size from designs
  height: 24px;
  padding: 0; // get rid of the browser padding
  ...
}
```

Reply:
Edit: Looks like those styles work in Firefox, but maybe not in chrome. It may take some more digging to find a solution for both, or Chrome may need some additional overrides for browser CSS.

Reply:
It might be worth looking at [-moz-color-swatch](https://developer.mozilla.org/en-US/docs/Web/CSS/::-moz-color-swatch) and -webkit-color-swatch as options to get the styles we want.

Reply:
Thanks, Matt for this really helpful feedback and great catches. The swatch and its container looks to render better in both chrome and firefox, but I'm still working on a few more issues with the component, such as the focus ring styling, and getting the aspect ratio to be 24x24 for the swatch, instead of 22x24 right now. I'll let you know when this is ready for review again.

Reply:
Thanks for the very helpful feedback @MattCheely 

I pushed up changes for this.